apps/condo/domains/meter/schema/MeterReading.test.js

```diff
@@ -1984,4 +1986,273 @@ describe('MeterReading', () => {
             )
         })
     })
+
+    describe('Manipulating readings taking into account MeterReportingPeriod for corresponding property/organization', () => {
+        let organization, source, meter, property, reportingPeriod
```

Comment:
Each test operates the same variable `reportingPeriod`. This variable may have a random value if tests run in parallel.

Reply:
you're right, this turned out to be a bad practice. now i create separate entities for every test